samples/SnapshotConsoleSample/SnapshotConsoleSample.c

```diff
@@ -474,7 +569,7 @@ int main(int argc, char *argv[]) {
 			"of events and a full order book.\n"
 			"-------------------------------------------------------------------------------\n"
 			"Usage: SnapshotConsoleSample <server address> <event type> <symbol>\n"
-			"       [order_source] [" TIME_PARAM_SHORT_TAG " <DD-MM-YYYY>] "
+			"       [order_source] [" FROM_TIME_PARAM_SHORT_TAG " <DD-MM-YYYY>] [" TO_TIME_PARAM_SHORT_TAG " <DD-MM-YYYY>]" 
```

Comment:
This time format does not correspond to the format specified below.

Reply:
Done